### ISSUE TITLE
[Update Request]: NirSoft.NirCmd 2.87

### ISSUE THREAD

**Issue by KarbitsCode** [microsoft/winget-pkgs#174470]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

NirSoft.NirCmd

### Package Version

2.87

### Please describe the changes you would like to see

There is a new release (2.87) available.
Download Page Url: https://www.nirsoft.net/utils/nircmd.html

**Comment by Dragon1573** [comment]
> - Installer: <https://www.nirsoft.net/utils/nircmd-x64.zip>
>   - Scan request id: `863dc056-3b9d-4ee6-9a50-ade320beb3b6`
>   - Error information: **BlockingDetectionFound**
> - Installer: <https://www.nirsoft.net/utils/nircmd.zip>
>   - Scan request id: `58bde5db-41e0-4750-aceb-aab02dab8a18`
>   - Error information: **BlockingDetectionFound**
>
> _Originally posted in https://github.com/microsoft/winget-pkgs/pull/168124#issuecomment-2297412390_

Sorry @stephengillie, what it **BlockingDetectionFound** here?

**Comment by stephengillie** [comment]
I don't have confirmation, but the suspicion is that `BlockingDetectionFound` is actually a Defender scan failing the run. Defender either detects and interrupts the download, or detects and quarantines the file, leading to the `0x80004005 Resource Missing` error code. It might be that a quarantine results in the expected https://github.com/microsoft/winget-pkgs/labels/Validation-Defender-Error label, while an interrupted download could result in `BlockingDetectionFound`.

This suspicion, combined with consistent reports that the host isn't blocking the download, has led me to believe that `BlockingDetectionFound` isn't actually detecting some kind of network block.

**Comment by sitiom** [comment]
A scan of the files in Windows Defender doesn't show anything:

![Image](https://github.com/user-attachments/assets/1d7a3e8e-1243-484d-9557-eb1db28ccfe8)

**Comment by DandelionSprout** [comment]
Based on (among other threads) #290625, it seems there's some random group called Sophos who complain about, claiming it has a, quote-unquote, "PUA reputation".

I have a decade's worth of experience in [properly peer-reviewed, semi-pro antimalware work](https://raw.githubusercontent.com/DandelionSprout/adfilt/refs/heads/master/Dandelion%20Sprout's%20Anti-Malware%20List.txt) to tell that it isn't PUA.

**Comment by KarbitsCode** [comment]
Yep, it's finally clear what is blocking these packages by getting https://github.com/microsoft/winget-pkgs/labels/Binary-Validation-Error label, from first time I saw the wingetbot finally report it (https://github.com/microsoft/winget-pkgs/pull/283355#issuecomment-3211442101)

Since this is just PUA, I wonder if that can be disputed though.

**Comment by KarbitsCode** [comment]
> Though all those very secretive and _very_-closed-source companies usually make it very difficult to figure out where to send them.

Exactly, at least Bitdefender was way easier and does have specific form for it, this one is just unclear.
